### PR TITLE
fixed division error bug

### LINKED ISSUE #15: Bug: Division by zero error on products
# CRITICAL ISSUE

When the client requests a single product, or all products, the status code is 500 and the response is always the following.

```
ZeroDivisionError at /products
division by zero
```

The most recent feature added by the team was adding an average rating key to each product. You might want to start there and see if there are any `try/catch` blocks you can implement.

### PR DESCRIPTION
**Pull request by andrew-webb07**
Description of PR that completes issue here...

## Changes

- bangazon/api/models/product.py

## Testing

Description of how to test code...

- [ ] Run migrations
- [ ] Run test suite
- [ ] Seed database

## Related Issues

- Fixes #15

### CODE REVIEW

File: bangazonapi/models/product.py [hunk 1]
```diff
@@ -62,8 +62,11 @@ def average_rating(self):
         for rating in ratings:
             total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
-        return avg
+        if total_rating == 0:
```
Comment (hannahhall) [inline bangazonapi/models/product.py]: The ticket wants the solution to be try/except block try that out instead of an if

**Review by hannahhall — CHANGES_REQUESTED** [review]

File: bangazonapi/models/product.py (hunk 1 above)
Comment (andrew-webb07) [reply bangazonapi/models/product.py]: try:
            avg = total_rating / len(ratings)
            return avg
            
        except ZeroDivisionError:
            if total_rating == 0:
                return 0

This look good?

Comment (hannahhall) [reply bangazonapi/models/product.py]: Yeah that works

**Review by hannahhall — APPROVED** [review]
👍

**Pull request merged**